dev-python/libvirt-python/libvirt-python-4.1.0.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 
-PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
```

Comment:
Please do the change in the unstable versions 4.2.0, 4.3.0 and 9999. You also have to do a revision bump, meaning rename 4.2.0 to 4.2.0-r1, and 4.3.0 to 4.3.0-r1 (keep 9999 as is).